packages/app/src/store/travel/index.ts

```
@@ -2,6 +2,19 @@ import { Coordinate } from "@model/map";
 import { TravelState } from "@model/travel";
 import { create } from "zustand";
 
+const DUMP_PATH = [
```

ESLint

[error] 'DUMP_PATH' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)